R/q.nfactors.R

```diff
@@ -0,0 +1,171 @@
+q.nfactors <- function(dataset, q.matrix = NULL, cutoff = NULL, siglevel = 0.05, quietly = FALSE, cor.method="pearson") {
```

Comment:
that's the wrapper function for all sorts of factor retention criteria. See documentation.